r2/r2/lib/pages/pages.py

```diff
@@ -294,6 +291,20 @@ class SideBoxPlaceholder(Wrapped):
     def __init__(self, node_id, link_text, link_path=None, sr_path=True):
         Wrapped.__init__(self, node_id=node_id, link_text=link_text, link_path=link_path, sr_path=sr_path)
 
+class MeetupsLink():
+    def render(_):
```

Comment:
Not entirely sure what you mean, but I think this is what you are looking for so that you can create a template:
`class MeetupsLink(Wrapped): pass`